mmv1/templates/terraform/constants/disk.tmpl

```diff
@@ -43,6 +43,18 @@ func IsDiskShrinkage(_ context.Context, old, new, _ interface{}) bool {
 	return new.(int) < old.(int)
 }
 
+func matchImageLink(old string) ([]string) {
```

Comment:
Since we're in a dedicated function now, should we just return the values (and possibly a success/failure boolean) directly rather than just return the array? That'll be clearer to callers.